### PR TITLE
Polish translation

### PR DESCRIPTION
**Pull request by kodiultimate**

### CODE REVIEW

**Comment by host505** [comment]
I see some Exodus Redux references in your file.
Also some msgid's don't match with the ones from the English language file (they must match, whether they have a msgstr definition or not).
If you fix these I'll merge.

File: resources/language/resource.language.pl_pl/strings.po [hunk 1]
```diff
@@ -0,0 +1,1088 @@
...
+msgid "Are you sure?"
+msgstr "Jesteś Pewien?"
+
+msgctxt "#32556"
+msgid "[B]TheOath[/B] : Library"
+msgstr "[B]TheOath[/B] : Biblioteka"
+
+msgctxt "#32557"
+msgid "[B]LIBRARY[/B] : Settings"
+msgstr "[B]ΒΙΒΛΙΟΘΗΚΗ[/B] : Ustawienia"
```
Comment (host505) [inline resources/language/resource.language.pl_pl/strings.po]: That's Greek :)

Comment (kodiultimate) [reply resources/language/resource.language.pl_pl/strings.po]: My bad :( I was using Greek strings.po as a base lol must have missed it

Comment (host505) [reply resources/language/resource.language.pl_pl/strings.po]: Ok, haven't merge yet, please fix so I can do so

**Comment by host505** [comment]
Thx

**Pull request merged**